lib/src/screens/connect_device/connect_device_page.dart

```diff
@@ -247,11 +261,27 @@ class ConnectDevicePageBodyState extends State<ConnectDevicePageBody> {
                       ),
                     )
                     .toList(),
-              ]
+              ],
+              if (widget.allowChangeWallet) ...[
+                PrimaryButton(
+                  text: S.of(context).wallets,
+                  color: Theme.of(context).extension<WalletListTheme>()!.createNewWalletButtonBackgroundColor,
+                  textColor: Theme.of(context).extension<WalletListTheme>()!.restoreWalletButtonTextColor,
+                  onPressed: _onChangeWallet,
+                )
+              ],
             ],
           ),
         ),
       ),
     );
   }
+
+  void _onChangeWallet() {
+    Navigator.of(context).pushNamed(
+      Routes.walletList,
+      arguments: (BuildContext context) =>
+          Navigator.of(context).pushNamed(Routes.dashboard),
```

Comment:
shouldn't this be `pushAndRemoveUntil`?

Reply:
the second one, push to dashboard, so that after the user goes to the dashboard, if he tapped back, he would get out of the app not go back to the wallet list page